docs/intro.txt

```diff
@@ -15,6 +15,15 @@ and Windows. To install, run:
             The user will be prompted for their login credentials when making API calls such as calling the ``process_circuits`` method or querying ``backend_info``.
 
 
+``pytket-quantinuum`` provides calendar visualization capabilities. The ``calendar`` extra-install-argument must be specified to install matplotlib and pandas.
+
+::
+
+    pip install pytket-quantinuum["calendar"]
```

Comment:
I don't think we want quotes here?

Reply:
should be around pytket-quantinuum. I will change

Reply:
> should be around pytket-quantinuum. I will change

Not needed around pytket-quantinuum either.

Reply:
OK. In the past, I ran into issues not having the quotes, but maybe this is because of zsh shell.